src/index.html

```diff
@@ -174,6 +189,53 @@ <h2 class="modal__title">New Post</h2>
           <p class="modal__preview-caption">""</p>
         </div>
       </div>
+      <div class="modal" id="avatar-open-modal">
+        <div class="modal__container">
+          <button
+            aria-label="Close Button"
+            type="button"
+            class="modal__close-button"
+          ></button>
+          <h2 class="modal__title">Edit Avatar</h2>
+          <form class="modal__form" id="edit-avatar-form-modal" novalidate>
+            <label for="profile-avatar-input" class="modal__label"
+              >Avatar
+              <input
+                id="profile-avatar-input"
+                required
+                type="link"
```

Comment:
`type="link"`

* Please, add `type="url"` for the `avatar` input, to validate it as a `link`